### ISSUE TITLE
About getting a poor result

### ISSUE THREAD

**Issue by XieZhiyao21** [leofansq/CPCL#6]
I tried to reproduce your project, but due to hardware limitations such as GPU video memory, I got a very bad result (mIoU of 55%), which is far from the result in your paper (76%). I was experimenting with 1/8 partitioning. I would like to know what are the possible reasons for this?
Due to the limited GPU memory, I changed the batchsize to 4 and the learning rate to 0.0025, both of which are half of the original configuration, and changed the parameter "pin_memory" from "True" to "False". No other changes have been made. My GPU is a 1080Ti.

Thank you!

**Comment by leofansq** [comment]
Both of the batchsize and learning rate are important hyper-parameters, and the performace highly depends on that.

**Comment by XieZhiyao21** [comment]
No other measures can deal with it?

**Comment by XieZhiyao21** [comment]
> Both of the batchsize and learning rate are important hyper-parameters, and the performace highly depends on that.

Sorry.I forget that I also change the size of picture into 256*256.Because I don't have enough GPU memory.
